xml/System.ComponentModel/VersionConverter.xml

```diff
@@ -33,7 +33,7 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Initializes a new instance of the <see cref="T:System.ComponentModel.VersionConverter" /> class.</summary>
```

Comment:
The cref here would generate a self reference. In other words, a hyperlink to the same page, which would refresh it, so it doesn't add any value. How does this sound instead?:

```suggestion
        <summary>Initializes a new version converter instance.</summary>
```

Reply:
I'd leave as-is. She's following the appropriate guidelines https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Constructor
And it's not really a self-reference since the constructor is in a different page than the class itself.

Reply:
OK then, keeping as is.

Reply:
Ah, my mistake. You're right, thanks for the clarification.